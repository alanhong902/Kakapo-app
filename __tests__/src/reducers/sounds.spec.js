import { path, prop } from 'ramda';
import configureStore from 'stores/configureStore';
import { soundActions } from 'actions/';
import { kakapoRes } from '../helper';

const store = configureStore();

const currState = () => prop('sounds', store.getState());
const windSound = () => path(['wind'], currState());

beforeEach(() => {
  localStorage.setItem('version', false);
  localStorage.removeItem('sounds');

  fetch.mockResponses([JSON.stringify(kakapoRes), { status: 200 }]);
});

test('[reducer/sounds] init sounds', () => {
  store.dispatch(soundActions.requestSuccess(kakapoRes));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] toggle play `on`', () => {
  store.dispatch(soundActions.play(windSound()));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] toggle play `off`', () => {
  store.dispatch(soundActions.play(windSound()));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] volume', () => {
  store.dispatch(soundActions.volume(windSound(), 0.25));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] edit', () => {
  const newData = { tags: 'newTag' };
  store.dispatch(soundActions.edit(windSound(), newData));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] remove', () => {
  store.dispatch(soundActions.remove(windSound()));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] reset', () => {
  store.dispatch(soundActions.reset(true));
  expect(currState()).toMatchSnapshot();
});

test('[reducer/sounds] clear', () => {
  store.dispatch(soundActions.reset());
  expect(currState()).toMatchSnapshot();
});
