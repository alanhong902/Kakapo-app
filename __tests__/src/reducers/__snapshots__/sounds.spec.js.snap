// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[reducer/sounds] clear 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
  "wind": Object {
    "editing": false,
    "file": "wind",
    "img": "wind",
    "name": "Wind",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, open, outdoors",
    "volume": 0.5,
  },
}
`;

exports[`[reducer/sounds] edit 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
  "wind": Object {
    "editing": true,
    "file": "wind",
    "img": "wind",
    "name": "Wind",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "newTag",
    "volume": 0.25,
  },
}
`;

exports[`[reducer/sounds] init sounds 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
  "wind": Object {
    "editing": false,
    "file": "wind",
    "img": "wind",
    "name": "Wind",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, open, outdoors",
    "volume": 0.5,
  },
}
`;

exports[`[reducer/sounds] remove 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
}
`;

exports[`[reducer/sounds] reset 1`] = `Object {}`;

exports[`[reducer/sounds] toggle play \`off\` 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
  "wind": Object {
    "editing": false,
    "file": "wind",
    "img": "wind",
    "name": "Wind",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, open, outdoors",
    "volume": 0.5,
  },
}
`;

exports[`[reducer/sounds] toggle play \`on\` 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
  "wind": Object {
    "editing": false,
    "file": "wind",
    "img": "wind",
    "name": "Wind",
    "playing": true,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, open, outdoors",
    "volume": 0.5,
  },
}
`;

exports[`[reducer/sounds] volume 1`] = `
Object {
  "birds": Object {
    "editing": false,
    "file": "birds",
    "img": "birds",
    "name": "Birds",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "animals, forest, outdoors, happy",
    "volume": 0.5,
  },
  "bonfire": Object {
    "editing": false,
    "file": "bonfire",
    "img": "bonfire",
    "name": "Bonfire",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "fire, outdoors, camping, cosy",
    "volume": 0.5,
  },
  "cat_purr": Object {
    "editing": false,
    "file": "cat_purr",
    "img": "cat",
    "name": "Cat Purring",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "pets, cat, indoor, content",
    "volume": 0.5,
  },
  "chimes": Object {
    "editing": false,
    "file": "chimes",
    "img": "chimes",
    "name": "Chimes",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "charm, outdoors, wind",
    "volume": 0.5,
  },
  "fan": Object {
    "editing": false,
    "file": "fan",
    "img": "fan",
    "name": "Fan",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, humming, indoors, repetitive",
    "volume": 0.5,
  },
  "leaves": Object {
    "editing": false,
    "file": "leaves",
    "img": "leaves",
    "name": "Leaves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, windy, outdoors",
    "volume": 0.5,
  },
  "paris_cafe": Object {
    "editing": false,
    "file": "paris_cafe",
    "img": "cafe",
    "name": "Paris Cafe",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, crowd, city, coffee, chatter",
    "volume": 0.5,
  },
  "paris_street_corner": Object {
    "editing": false,
    "file": "paris_street_corner",
    "img": "paris",
    "name": "Paris Street Corner",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "people, city, cars, streets, outdoors",
    "volume": 0.5,
  },
  "rain": Object {
    "editing": false,
    "file": "rain",
    "img": "rain",
    "name": "Rain",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, water, outdoors",
    "volume": 0.5,
  },
  "stream": Object {
    "editing": false,
    "file": "stream",
    "img": "water",
    "name": "Stream",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, water, running, outdoors",
    "volume": 0.5,
  },
  "thunder": Object {
    "editing": false,
    "file": "thunder",
    "img": "thunder",
    "name": "Thunder",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "storm, weather, nature, open, outdoors",
    "volume": 0.5,
  },
  "train": Object {
    "editing": false,
    "file": "train",
    "img": "train",
    "name": "Train",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "machine, travel, repetitive, soothing",
    "volume": 0.5,
  },
  "waves": Object {
    "editing": false,
    "file": "waves",
    "img": "waves",
    "name": "Waves",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "ocean, nature, open, water",
    "volume": 0.5,
  },
  "wind": Object {
    "editing": false,
    "file": "wind",
    "img": "wind",
    "name": "Wind",
    "playing": false,
    "progress": 1,
    "recentlyDownloaded": false,
    "source": "file",
    "tags": "weather, nature, open, outdoors",
    "volume": 0.25,
  },
}
`;
